lms/djangoapps/commerce/admin.py

```diff
@@ -0,0 +1,8 @@
+""" Admin site bindings for commerce app. """
+
+from django.contrib import admin
+
+from commerce.models import CommerceConfiguration
+from config_models.admin import ConfigurationModelAdmin
+
+admin.site.register(CommerceConfiguration, ConfigurationModelAdmin)
```

Comment:
Can you please be more specific which values should be displayed in the admin panel. This is what I currently see: http://i.imgur.com/exUJHyr.png